plugins/jobtype/src/azkaban/jobtype/HadoopHiveJob.java

```diff
@@ -22,12 +22,15 @@
 import java.io.IOException;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
```

Comment:
I don't think that exercises the cancel method, so jobs are not going to be killed.
Should we be handling that scenario?